hive-metastore/src/main/java/org/apache/iceberg/hive/HiveOperationsBase.java

```diff
@@ -76,11 +138,11 @@ default boolean exposeInHmsProperties() {
     return maxHiveTablePropertySize() > 0;
   }
 
-  default void setSchema(TableMetadata metadata, Map<String, String> parameters) {
```

Comment:
we need to keep the original method (for backwards compatibiliy), but the method can refer to the new `setSchema()` method:

```
default void setSchema(TableMetadata metadata, Map<String, String> parameters) {
    setSchema(metadata.schema(), parameters);
  }
```